modules/020-deckhouse/docs/FAQ.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "The deckhouse module: FAQ"
+---
+
+## How to run kube-bench in my cluster?
+
+First, you have to exec in Deckhouse Pod:
+```shell
+kubectl -n d8-system exec -ti deploy/deckhouse -- bash
+```
+
+Then you have to select which node you want to run kube-bench.
+
+* Run on random node:
+  ```shell
+  curl -s https://raw.githubusercontent.com/aquasecurity/kube-bench/main/job.yaml | kubectl create -f -
+  ```
+  
+* Run on specific node, e.g. control-plane node:
+  ```shell
+  curl -s https://raw.githubusercontent.com/aquasecurity/kube-bench/main/job.yaml | yq r - -j | jq '.spec.template.spec.tolerations=[{"operator": "Exists"}] | .spec.template.spec.nodeSelector={"node-role.kubernetes.io/control-plane": ""}' | kubectl create -f -
```

Comment:
I thought some mounts were required to execute this job on a master node successfully. Do you remember something about it?

Reply:
[Job](https://raw.githubusercontent.com/aquasecurity/kube-bench/main/job.yaml) has default `volumes` and `volumeMounts` that match with Deckhouse directories. It's no need to additionally manage volumes because everything just works fine out of the box.
```
      volumes:
        - name: var-lib-etcd
          hostPath:
            path: "/var/lib/etcd"
        - name: var-lib-kubelet
          hostPath:
            path: "/var/lib/kubelet"
        - name: var-lib-kube-scheduler
          hostPath:
            path: "/var/lib/kube-scheduler"
        - name: var-lib-kube-controller-manager
          hostPath:
            path: "/var/lib/kube-controller-manager"
        - name: etc-systemd
          hostPath:
            path: "/etc/systemd"
        - name: lib-systemd
          hostPath:
            path: "/lib/systemd"
        - name: srv-kubernetes
          hostPath:
            path: "/srv/kubernetes"
        - name: etc-kubernetes
          hostPath:
            path: "/etc/kubernetes"
        - name: usr-bin
          hostPath:
            path: "/usr/bin"
        - name: etc-cni-netd
          hostPath:
            path: "/etc/cni/net.d/"
        - name: opt-cni-bin
          hostPath:
            path: "/opt/cni/bin/"
```